app/server.js

```diff
@@ -23,7 +20,7 @@ async function createServer () {
         options: {
           host: config.redisHost,
           port: config.redisPort,
-          partition: 'mine-support'
+          partition: config.redisPartition
```

Comment:
The partition is now passed in as a parameter to prevent key collision between deployments. 
However I don't think there's anything currently in place to prevent collision between user session.